Makefile

```diff
@@ -52,21 +51,15 @@ test/%.bats: ALWAYS_RUN
 
 lint: eclint hadolint shellcheck
 
+HADOLINT_VERSION="2.4.1"
```

Comment:
I prefer to have the variables on the top of the `Makefile`, but this may be fine. However, why do you quote the value(s)? This way, `""` becomes part of the value which is not desired in this case :)

Reply:
You're right, sorry, I forgot to finalize this last night